source/_data/glossary.yml

```
@@ -218,6 +218,13 @@
     or RGB color control.
   link: /integrations/light
 
+- term: Matter
+  definition: >-
+    Matter is an open-source standard that defines how to control smart home devices on a WiFi or Thread network. The aim of the standard is to improve security and to make devices interoperable across vendors, replacing proprietary protocols for smart home ecosystems. Unlike other standards, Matter allows joining the same device to multiple controllers. For example, you can add a light to Google Home, Apple Home, and Home Assistant at the same time. A bridge device can be used to connect devices running on other smart home technologies such as Zigbee or Z-Wave.
```

<https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>

```suggestion
    Matter is an open-source standard that defines how to control smart home devices on a Wi-Fi or Thread network. The aim of the standard is to improve security and to make devices interoperable across vendors, replacing proprietary protocols for smart home ecosystems. Unlike other standards, Matter allows joining the same device to multiple controllers. For example, you can add a light to Google Home, Apple Home, and Home Assistant at the same time. A bridge device can be used to connect devices running on other smart home technologies such as Zigbee or Z-Wave.
```